homeassistant/helpers/device_registry.py

```diff
@@ -18,6 +18,7 @@
 STORAGE_VERSION = 1
 SAVE_DELAY = 10
 
+CONNECTION_LOCAL_DEVICE = 'local'
```

Comment:
This is not being used.

Reply:
That was from my first attempt to add a device for the z-wave stick and set up `via_hub` - would you prefer that I go ahead and remove that and merge without `via_hub` currently, or should I make another attempt at getting it added in?